packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -479,7 +496,7 @@ const Breakouts = WebexPlugin.extend({
             locusUrl: this.locusUrl,
           },
         })
-        .catch((err) => {
+        .catch((err: unknown) => {
```

Comment:
I would like to set `unknown` everywhere in this PR, but there are cases where `err` is explicitly an object. With object handling, the only way to handle it is to set it to `any` due to the lack of error handlers for errors (see screenshot). In the future, it would be great to create a proper Error type structure and align it accordingly.
<img width="873" alt="Screenshot 2024-06-25 at 17 43 09" src="https://github.com/webex/webex-js-sdk/assets/38460776/bb13f6c4-94dc-44dc-a9e5-412e57b51fd1">